firefox-ios/Client/Frontend/Settings/AppearanceSettings/Zoom/ZoomSiteListView.swift

```diff
@@ -8,26 +8,62 @@ import Storage
 
 struct ZoomSiteListView: View {
     let theme: Theme
-    let zoomLevels: [DomainZoomLevel]
+    @Binding var domainZoomLevels: [DomainZoomLevel]
+    let onDelete: (IndexSet) -> Void
 
-    var textColor: Color {
-        return theme.colors.textPrimary.color
+    private struct UX {
+        static var sectionPadding: CGFloat { 16 }
```

Comment:
Out of curiosity, why not `=`?
```suggestion
        static var sectionPadding: CGFloat = 16 
```

Reply:
Copy-paste mainly for the ones using `{ }` and `=` for the manually created. I will update to = for all